scripts.d/ta/825_ha_mgmt_ip.sh

```diff
@@ -19,7 +19,14 @@ for WEKA_CONTAINER in $(sudo weka local ps --output name --no-header | grep -e c
         if [[ -n ${NET_NAME} ]]; then
             if [[ $(ip -4 -j -o addr show dev ${NET_NAME} 2>/dev/null | tr -d \"\[:blank:]) =~ "local:"([0-9\.]+) ]]; then
                 NET_IP=${BASH_REMATCH[1]}
-                if [[ ! $(weka local resources -C ${WEKA_CONTAINER} --stable | grep -e ^"Management IPs") =~ ${NET_IP} ]]; then
```

Comment:
This is minor, and not very likely, but if
NET_IP=192.168.123.1   and Management IPs=192.168.123.11

you'll get a false match here. It's plausible someone could typo those, and that's not an easy thing to spot with ye olde eyeballs.

Reply:
Yeah, sorry, this should already be addressed in the second commit.

Reply:
Bah, Sorry, I thought i *was* looking at the most recent.